mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,42 @@
+interface CheckboxProps {
+  /**
+   * Status flag whether the element is checked or not
+   */
+  checked: boolean;
+  /**
+  * Label's text
+  */
+  label: string;
+  /**
+   * Additional custom class name for the component
+  */
+  className?: string;
+  /**
+   * Select`s value
+  */
+  value: string;
+  /**
+   * Select's name
+  */
+  name: string;
+  /**
+   * The flag sets the element to be required
+   */
+  required?: boolean;
+  /**
+   * Callback triggered on checkbox or label click
+   */
+  onCheckedChange: () => void;
+}
+
+const Checkbox = (props : CheckboxProps) => {
+  const {className} = props;
+
+  return (
+    <label className={className}>
+      <input type="checkbox" />
+    </label>
+  );
+};
+
+export {Checkbox};
```

Comment:
does linter works? @Ekaterina1994 why there is no comment for public member?

Reply:
how to use this component?

Reply:
PR with eslint doesn't mereged yet

Reply:
corrected following the example of other components